designer/client/src/components/table/SearchFilter.tsx

```diff
@@ -9,13 +10,41 @@ const flex = css({
     flex: 1,
 });
 
+export function AdvancedOptionsIcon(props: {
+    isActive?: boolean;
+    collapseHandler: React.Dispatch<React.SetStateAction<boolean>>;
+}): JSX.Element {
+    const theme = useTheme();
+
+    const toggleCollapseHandler = () => {
+        props.collapseHandler((p) => !p);
+    };
+
+    return (
+        <AdvancedSearchSvg
```

Comment:
can we add a cursor pointer for the icon, to indicate that it's a clickable element?